updatezinfo.py

```diff
@@ -17,9 +18,19 @@ def main():
 
     if not os.path.isfile(metadata['tzdata_file']):
         print("Downloading tz file from iana")
-        request.urlretrieve(os.path.join(metadata['releases_url'],
-                                         metadata['tzdata_file']),
-                            metadata['tzdata_file'])
+        for x in range(1, 6):
+            try:
+                request.urlretrieve(os.path.join(metadata['releases_url'],
+                                                 metadata['tzdata_file']),
+                                    metadata['tzdata_file'])
+                break
+            except Exception:
```

Comment:
I took the path of "whatever happens retry" but I guess its better to be more specific :-P

I double checked the docs for urllib.request and yeah. Seems like `URLError` is the one